awx/main/isolated/manager.py

```diff
@@ -74,6 +74,7 @@ def build_runner_params(self, hosts, verbosity=1):
         env['ANSIBLE_RETRY_FILES_ENABLED'] = 'False'
         env['ANSIBLE_HOST_KEY_CHECKING'] = str(settings.AWX_ISOLATED_HOST_KEY_CHECKING)
         env['ANSIBLE_LIBRARY'] = os.path.join(os.path.dirname(awx.__file__), 'plugins', 'isolated')
+        env['ANSIBLE_COLLECTIONS_PATHS'] = settings.INVENTORY_COLLECTIONS_ROOT
```

Comment:
We should rename this variable now that it's being used for more than just inventory plugins.

Reply:
It's in the name of the folder too, which seems like a bigger problem than the setting name.

My thinking is these will be replaced before it really matters.

Reply:
Renaming a few things isn't hard. We dont know if / when this vendoring solution is going away. Shouldn't we spend a few extra seconds to make things make sense?

Reply:
Why not just make the variable `settings.AWX_ANSIBLE_COLLECTIONS_PATHS` so it's super clear what it represents?

Reply:
going with Ryan's suggestion, thanks!